package.json

```diff
@@ -127,20 +127,20 @@
     "react-native-linear-gradient": "^2.6.2",
     "react-native-pager-view": "6.1.4",
     "react-native-progress": "bluesky-social/react-native-progress",
-    "react-native-reanimated": "^3.4.2",
+    "react-native-reanimated": "~3.3.0",
```

Comment:
Slight downgrade, need to test this

Reply:
Seems fine on all platforms

Reply:
Why was this downgraded?

Reply:
If its because of expo v49 compatibility, that only matters in the managed workflow afaik. We can use versions that aren't directly compatible with our version of expo because we use `prebuild` and build the app. 

Specifically this reanimated update was to fix a bug #1198 which was a fatal crash issue

Reply:
Oh good call I forgot about that. Yeah we'll just ignore this one then 👍

Reply:
Done